Source/Core/Core/Slippi/SlippiUser.cpp

```diff
@@ -145,15 +145,21 @@ void SlippiUser::OpenLogInPage()
 {
 #ifdef _WIN32
 	std::string folderSep = "%5C";
+#elif defined(__APPLE__)
```

Comment:
I think it might be best to just set `folderSep` to "/" and then leave the rest of the function the same? That way backslashes will get converted to forward slashes. Though I think on mac there shouldn't be backslashes in the path variable.

Reply:
I guess maybe that's not a great idea since you can have backslashes in file names/folder names on Mac. I guess that might also be an issue on Linux...

Reply:
I actually addressed these comments myself, changing the upstream branch, you can just pull the changes in?

Reply:
Hmm actually I can't change the base, just pull in the pr/153 branch

Reply:
I moved your `SlippiUser::OpenLogInpage()` over manually, pulling in seemed to cause conflicts that I don't have time to figure out tonight.